manifests/init.pp

```diff
@@ -57,9 +66,11 @@
 # $use_sudoersd::               Add a file to /etc/sudoers.d (true) or uses augeas (false)
 #                               type:boolean
 #
-# $puppetca::                   Use Puppet CA
+# $puppetca::                   Enable Puppet CA proxy
```

Comment:
Nitpick, using the word 'proxy' here could be misleading - it's a smart proxy module or feature, rather than a Puppet CA proxy server.